src/kernel/arch/x86/boot.zig

```diff
@@ -1,11 +1,10 @@
-const constants = @import("constants.zig");
-
 const ALIGN = 1 << 0;
 const MEMINFO = 1 << 1;
 const MAGIC = 0x1BADB002;
 const FLAGS = ALIGN | MEMINFO;
 
-const KERNEL_PAGE_NUMBER = constants.KERNEL_ADDR_OFFSET >> 22;
+const KERNEL_ADDR_OFFSET: u32 = 0xC0000000;
```

Comment:
Why did these have to be moved from constants.zig?

Reply:
I removed it bc I needed the offset in the tty, and as the constants.zig was in the x86, it was difficult to import this file. But now I know how to use addPackagePath, i can add this back

Reply:
Makes sense :)